package.json

```diff
@@ -2,10 +2,11 @@
   "name": "exp",
   "version": "0.0.0",
   "private": true,
+  "type": "module",
   "scripts": {
-    "start": "node ./server.js",
-    "dev": "nodemon ./server.js",
-    "eslint": "eslint ."
+    "start": "node --experimental-specifier-resolution=node ./server.js",
```

Comment:
Ciekawostka w node v20.5.0 twój kod przez tą flagę nie działa. Możesz sobie dodać tutaj nad skryptami
```
    "engines": {
        "node": "18.17.1"  // czy inną wersję której używasz
    }
```
Zaoszczędza to 20 min szukania co jest nie tak z kodem :smiling_face_with_tear: